src/main/resources/theme/mod_manager.fxml

```diff
@@ -1,46 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.geometry.Insets?>
-<?import javafx.scene.control.Button?>
-<?import javafx.scene.control.ListView?>
-<?import javafx.scene.control.ToggleButton?>
-<?import javafx.scene.control.ToggleGroup?>
-<?import javafx.scene.control.Tooltip?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.VBox?>
-<VBox xmlns:fx="http://javafx.com/fxml/1" fx:id="root" xmlns="http://javafx.com/javafx/11.0.1"
-      fx:controller="com.faforever.client.mod.ModManagerController">
+<?import javafx.geometry.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<VBox fx:id="root" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.faforever.client.mod.ModManagerController">
     <!-- Mods -->
     <HBox fillHeight="false" spacing="10.0">
-        <ToggleButton fx:id="uiModsButton" maxWidth="1.7976931348623157E308" minWidth="-Infinity"
-                      mnemonicParsing="false" onAction="#onShowUIMods" styleClass="game-list-type"
-                      text="%mod.manager.uiMods" HBox.hgrow="ALWAYS">
+        <ToggleButton fx:id="uiModsButton" maxWidth="1.7976931348623157E308" minWidth="-Infinity" mnemonicParsing="false" onAction="#onShowUIMods" styleClass="game-list-type" text="%mod.manager.uiMods" HBox.hgrow="ALWAYS">
             <tooltip>
-                <Tooltip text="%mod.manager.uiModsTooltip"/>
+                <Tooltip text="%mod.manager.uiModsTooltip" />
             </tooltip>
             <toggleGroup>
-                <ToggleGroup fx:id="viewToggleGroup"/>
+                <ToggleGroup fx:id="viewToggleGroup" />
             </toggleGroup>
         </ToggleButton>
-        <ToggleButton fx:id="simModsButton" maxWidth="1.7976931348623157E308" minWidth="-Infinity"
-                      mnemonicParsing="false" onAction="#onShowSimMods" selected="true" styleClass="game-list-type"
-                      text="%mod.manager.simMods" toggleGroup="$viewToggleGroup" HBox.hgrow="ALWAYS">
+        <ToggleButton fx:id="simModsButton" maxWidth="1.7976931348623157E308" minWidth="-Infinity" mnemonicParsing="false" onAction="#onShowSimMods" prefWidth="152.0" selected="true" styleClass="game-list-type" text="%mod.manager.simMods" toggleGroup="$viewToggleGroup" HBox.hgrow="ALWAYS">
             <tooltip>
-                <Tooltip text="%mod.manager.simModsTooltip"/>
+                <Tooltip text="%mod.manager.simModsTooltip" />
             </tooltip>
         </ToggleButton>
```

Comment:
The UI and SIM buttons should be the same size

Reply:
fixed